about.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>PCD</title>
+  </head>
+  <body >
+    <div id="navLanding" class="aboutPage dFlex">
+      <nav>
+        <img class="noneDesktop" id="hamburger" src="assets/mobileMenu.png" alt="hamburger menu">
+        <div class="noneMobileToFlex bgDark navMedia">
+          <a href="https://angel.co" target="_blank" rel="noreferrer noopener"><img src="assets/angellist.svg" alt="angel list"></a>
+          <a href="https://twitter.com/ProcessingOrg" target="_blank" rel="noreferrer noopener"><img src="assets/twitter.svg" alt="twitter"></a>
+          <a href="#" class="textWhite fs12 decorNone" target="_blank" rel="noreferrer noopener">English</a>
+          <a href="#" class="textWhite fs12 decorNone" target="_blank" rel="noreferrer noopener">Register</a>
+          <a class="textWhite fs12 decorNone" href="https://processing.org/" target="_blank" rel="noreferrer noopener">Processing Org</a>
```

Comment:
- [ ] Kindly check on your design as the previous reviewer noted this section is supposed to be a logout button as shown in the design 
![image](https://user-images.githubusercontent.com/61408860/190178747-7f11459e-7761-45d4-9c6d-7675f11d8863.png)

Reply:
@Qoosim  @danmainah  Thank you for your time with the review. According to the project instructions this are the things you need to focus more of the design

> Colors.
>     Typography: font face, size and weight (we suggest using [Lato](https://www.latofonts.com/) as it is a free font similar to the one used in the original design).
>     Layout: composition and space between elements.

Also it is mentioned with should personalize the content of the page

> You should personalize the content of your page.

That is the reason I used other options instead of the logout and my Page options that the design navbar has. 
I will change them accordingly to your reviews, but I think its not a big deal for the project. Otherwise it would have been explained in the instructions to make the navbar exactly as the design.
I'm adding a screenshot of how the page currently is deployed.

![Screenshot (124)](https://user-images.githubusercontent.com/8645872/190187023-7677ba86-456f-4cc6-8599-229b0fd854b1.png)